### ISSUE TITLE
hold to right click (basic touch enhancment)

### ISSUE THREAD

**Issue by jlnxr** [minbrowser/min#236]
When using a touch screen in Chromium, holding a link for a few seconds triggers the rightclick menu. This feature should be included in min to help with use on touchscreen computers, as it would come without any detriment or change in function to non touch screen users. 

Another option would be the ability to 'grab' links and drag them to the tab bar to open a new tab, which is what firefox does, although that only helps to open a link link in a new tab, not to right click, and is less natural then hold to right click.

**Comment by AluminiumTech** [comment]
I don't think this is supported by the underlying technology behind the browser since Electron isn't Hi-DPI aware either.
